bedrock/firefox/views.py

```diff
@@ -526,3 +527,51 @@ def ios_testflight(request):
     return l10n_utils.render(request,
                              'firefox/testflight.html',
                              {'newsletter_form': newsletter_form})
+
+
+class FirefoxProductDesktopView(BlogPostsView):
+    blog_posts_limit = 3
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
+
+    def get_template_names(request):
+        locale = l10n_utils.get_locale(request)
+
+        if locale == 'en-US':
+            template_name = 'firefox/products/desktop.html'
+        else:
+            template_name = 'firefox/desktop/index.html'
+
+        return [template_name]
+
+
+class FirefoxProductAndroidView(BlogPostsView):
+    blog_posts_limit = 3
+    blog_posts_template_variable = 'articles'
+    blog_slugs = 'firefox'
```

Comment:
Hmm, I see they don't have categories with these tags ready just yet. Maybe we could just add a comment hinting what they should be once ready?